### ISSUE TITLE
Crash in FilterDetail.applyFilter()

### ISSUE THREAD

**Issue by leberwurstsaft** [FlexMonkey/Filterpedia#1]
I was looking through all the filters, when a crash happened in Triangle Kaleidoscope.
Interestingly, it didn't happen again on next launch. Then I noticed that some of the filters above Triangle Kaleidoscope now showed different output. Somehow some of them didn't show output at all before, but I don't remember which (I think the [...] Tile filters). I can also not reproduce this.

```
2016-02-05 19:37:32.750 Filterpedia[1538:582807] -[CIVector floatValue]: unrecognized selector sent to instance 0x14c65fd30
2016-02-05 19:37:32.751 Filterpedia[1538:582807] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[CIVector floatValue]: unrecognized selector sent to instance 0x14c65fd30'
```

![crash stacktrace](https://cloud.githubusercontent.com/assets/108899/12855566/85bddad4-cc40-11e5-8aab-ef589e37279b.png)
![crash line](https://cloud.githubusercontent.com/assets/108899/12855571/8ca73b38-cc40-11e5-9407-a796026bfb41.png)

**Comment by FlexMonkey** [comment]
Hmmm - I can't reproduce either. I wonder if a previous filter had set a parameter of the same name with values that crashed _Triangle Kaleidoscope_ - I'll keep this open and keep hammering it :)

**Comment by leberwurstsaft** [comment]
Yes, probably some "cross talking" like that. Could be the same reason for some of the other filters not really showing anything, or so it seemed. That was also fixed when I tried again (but obviously didn't fiddle with all the sliders in the same way again).
